Owncloud iOs Client/Login/Login/LoginViewController.m

```diff
@@ -1952,17 +1965,20 @@ - (void) checkIfServerAutentificationIsNormalFromURL {
     
     [[AppDelegate sharedOCCommunication] checkServer:_connectString onCommunication:[AppDelegate sharedOCCommunication] successRequest:^(NSHTTPURLResponse *response, NSString *redirectedServer) {
         
-        BOOL isInvalid = NO;
-        isLoginButtonEnabled = YES;
-        
         //Update the interface depend of if isInvalid or not
-        if (isInvalid) {
+
+        if (k_is_sso_active) {
```

Comment:
Maybe you can change this if with:
isLoginButtonEnabled = k_is_sso_active
hasInvalidAuth = !k_is_sso_active